packages/react-native-editor/CHANGELOG.md

```diff
@@ -13,7 +13,10 @@ For each user feature we should also add a importance categorization label  to i
 
```

Comment:
Maybe we can put a 1.39.1 section here and update on the change included in the 1.39.1 betafix.